### ISSUE TITLE
Run only once, including a returning page visit

### ISSUE THREAD

**Issue by bramvandenbulcke** [alexmacarthur/typeit#190]
It's currently possible to control the loop functionality when the visitor stays on the page. But if a visitor that continues browsing the website, visiting other pages, returns to the page with the TypeIt animation, the animation starts over again. 

Is it possible to run the animation only once, for a specific user on the website?

**Comment by alexmacarthur** [comment]
Hi! This is something pretty far out of scope of TypeIt itself. To pull this off, you'd need to store whether it's been viewed locally somehow (like localStorage or sessionStorage, maybe), and then conditionally initiate TypeIt if you can determine that it's not been viewed. It's totally doable, just not something TypeIt itself would manage.
